makefiles/libnet.mk

```diff
@@ -0,0 +1,45 @@
+ifneq ($(PROCURSUS),1)
+$(error Use the main Makefile)
+endif
+
+SUBPROJECTS    += libnet
+LIBNET_VERSION := 1.2
+DEB_LIBNET_V   ?= $(LIBNET_VERSION)
+
+libnet-setup: setup
+	wget -q -nc -P $(BUILD_SOURCE) https://github.com/libnet/libnet/releases/download/v$(LIBNET_VERSION)/libnet-$(LIBNET_VERSION).tar.gz
```

Comment:
Use `GITHUB_ARCHIVE` function. This will download the source code tar ball generated by github (as opposed to one manually attached in releases)